composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "5.0.x",
+        "illuminate/support": "5.x",
```

Comment:
Can you change this into: "illuminate/support": "5.0.x|5.1.x" and merge it? 
I would like to use it with laravel 5.1
Thanks.